
                int func(){
                    return 1;
                }

                void main(){
                    func(func());
                    // We have an error
                }
                